cloud_testing/mock_server/src/main/kotlin/ftl/Main.kt

```diff
@@ -83,7 +83,7 @@ object Main {
                 // GET /toolresults/v1beta3/projects/delta-essence-114723/histories/1/executions/1/steps/1
                 get("/toolresults/v1beta3/projects/{projectId}/histories/{historyId}/executions/{executionId}/steps/{stepId}") {
                     println("Responding to GET ${call.request.uri}")
-                    val projectId = call.parameters["projectId"]
+                    // val projectId = call.parameters["projectId"]
```

Comment:
remove?

Reply:
I left it in there for now because the mock server should be using projectId somewhere in the returned response.